Pinta.Effects/Classes/ErrorDiffusionMatrix.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace Pinta.Effects;
+
+public enum PredefinedDiffusionMatrices
+{
+	Sierra,
+	TwoRowSierra,
+	SierraLite,
+	Burkes,
+	Atkinson,
+	Stucki,
+	JarvisJudiceNinke,
+	FloydSteinberg,
+	FakeFloydSteinberg,
+}
+
+internal sealed class ErrorDiffusionMatrix
```

Comment:
Please also add some doc comments for this class

Reply:
I just added a doc comment with basic explanation